test/stablecoin_converter.js

```diff
@@ -20,7 +20,7 @@ contract("StablecoinConverter", async (accounts) => {
 
   const [user_1, user_2, user_3, solutionSubmitter] = accounts
   let BATCH_TIME
-  beforeEach(async () => {
+  before(async () => {
```

Comment:
With this change, you'd be creating the unit under test only once and reusing the same instance. This would kind of destroy the independence assumption between tests since state would be carried from one test to the other. Same argument for not reusing the mocks.